### DISCUSSION TITLE
post/rebus-azure-storage-queues

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#72] [Announcements]
# post/rebus-azure-storage-queues

Mark Heath's Development Blog

https://markheath.net/post/rebus-azure-storage-queues

**Comment by seankearon** [comment]
@Truebzyq - I am sorry that you have not had a good experience with Rebus.  I feel compelled to comment as your description of working with Rebus is very different to my experiences and I would not want readers of Mark's blog to think that what you describe is usual.  

I have used Rebus in production for many years with heavy workloads and find it to be a very high-quality library - both in terms of the code in production and also in terms of excellent developer-experience.  My experience of the project on GitHub is that is active, welcoming, and very well managed.
